src/rest/mormot.rest.mvc.pas

```diff
@@ -631,6 +635,7 @@   TMvcRunOnRestServer = class(TMvcRunWithViews)
       aViews: TMvcViewsAbstract = nil;
       aPublishOptions: TMvcPublishOptions=
         [low(TMvcPublishOption) .. high(TMvcPublishOption)]); reintroduce;
+    destructor Destroy; override;

```

Comment:
please add a short comment like
`/// finalize this MVC server logic instance`

Reply:
Done.